app/helpers/dashboard_helper.rb

```diff
@@ -21,7 +21,7 @@ def dashboard_cards_by_role
       ],
       evaluator: [
         { image_path: 'star_half', href: 'evaluations',
-          alt: 'submissions and evaluations', title: 'Submissions & Evaluations',
+          alt: 'submissions and evaluations', title: 'Evaluations',
           subtitle: 'View submissions assigned to me and provide evaluations.' },
```

Comment:
Figma was just updated, can you adjust the copy
```suggestion
          subtitle: 'View and evaluate submissions assigned to me.' },
```